### PR TITLE
Added  actor tracing configuration flag

### PR DESCRIPTION
**Pull request by ad1622**
use -atcfg=<config> to turn off tracing for some things, for example -atcfg=mt:mp:pc to disable message timestamps, message parameters and promises.

### CODE REVIEW

File: src/tools/actors/ActorExecutionTrace.java [hunk 1]
```diff
@@ -275,7 +279,17 @@ public static void mailboxExecuted(final Mailbox m, final Actor actor) {
       b.put(Events.Mailbox.id);
       b.putLong(m.getBasemessageId()); //base id for messages
       b.putLong(actor.getActorId()); //receiver of the messages
+
       int idx = 0;
+      byte eventid = MESSAGE_BASE;
+
+      if (VmSettings.MESSAGE_TIMESTAMPS) {
```
Comment (smarr) [inline src/tools/actors/ActorExecutionTrace.java]: ok, you have to explain to me why this bit is set depending on the static configuration of the interpreter. This is not depending on a dynamic value. So, why is this info relevant in each event id?

(same comment below)

File: src/tools/actors/ActorExecutionTrace.java [hunk 2]
```diff
@@ -287,32 +301,36 @@ public static void mailboxExecuted(final Mailbox m, final Actor actor) {
           b.putShort((short) idx);
         }
 
-        if (em instanceof PromiseSendMessage) {
-          b.put(Events.PromiseMessage.id);
+        if (em instanceof PromiseSendMessage && VmSettings.PROMISE_CREATION) {
+          b.put((byte) (eventid | PROMISE_BIT));
```
Comment (smarr) [inline src/tools/actors/ActorExecutionTrace.java]: here it depends on run time info, but, it seems this bit wasn't there before. So, why is it needed now?

File: src/som/VmSettings.java [hunk 3]
```diff
@@ -33,6 +40,16 @@
     MEMORY_TRACING = getBool("som.memoryTracing",   false);
     DISABLE_TRACE_FILE = getBool("som.disableTraceFile", false);
 
+    //disabling of message or actor tracing in general is probably not a good idea and would require to turn off other tracing as well;
```
Comment (ad1622) [inline src/som/VmSettings.java]: The comment was intended to say, that turning off actor/message tracing would be as useful as no tracing at all.

File: src/tools/actors/ActorExecutionTrace.java (hunk 2 above)
Comment (ad1622) [inline src/tools/actors/ActorExecutionTrace.java]: when promise tracing is turned off, the promise message is treated like a normal message

**Pull request merged**